plugins/woocommerce/includes/class-wc-post-types.php

```diff
@@ -380,6 +380,7 @@ public static function register_post_types() {
 					'supports'        => false,
 					'capability_type' => 'product',
 					'rewrite'         => false,
+					'show_in_rest'    => true,
```

Comment:
Does this have any side-effects that would be unexpected?

Reply:
Good question, I was kinda curious about that to 🤷‍♂️ 
Last time we made changes here, we didn't find it out until after the code freeze and other people started complaining.